Packs/CiscoUmbrellaReporting/Integrations/CiscoUmbrellaReporting/CiscoUmbrellaReporting.py

```diff
@@ -0,0 +1,1138 @@
+from CommonServerPython import *
+from typing import Tuple, Callable
+from requests.auth import HTTPBasicAuth
+from datetime import datetime
+import requests
+
+# Disable insecure warnings
+requests.packages.urllib3.disable_warnings()
+
+# CORTEX XSOAR COMMAND CONSTANTS
+COUNT = 0
+LIMIT = 50
+FROM_DATE = "-7days"
+TO_DATE = "now"
+OFFSET = 0
+INTEGRATION_CONTEXT_NAME = 'UmbrellaReporting'
+TOKEN_ENDPOINT = "https://management.api.umbrella.com/auth/v2/oauth2/token"
+IP_PARAM = 'ip'
+DOMAIN_PARAM = 'domains'
+SHA256_PARAM = 'sha256'
+INTRUSION_ACTION = 'intrusion_action'
+DATE_TIME_FORMAT = "%b %d, %Y %H:%M %p"
```

Comment:
I'm not sure about this time format - I think it should be ISO format. Let's discuss that offline.

Reply:
We took the format from the cisco dashboard.

Reply:
We usually display timestamps in XSOAR with the following format: 
`DATE_FORMAT = '%Y-%m-%dT%H:%M:%SZ'  # ISO8601 format with UTC, default in XSOAR`

If this constant determines the time format of timestamps which are displayed in the readable output of commands or in the context, I think it would be better to use the ISO8601. 
@meital-aizen - what do you think?

Reply:
@ShacharKidor You're right, this is the format we usually work with.
@RamboV Please modify.

Reply:
@RamboV - Please change that.

Reply:
@ShacharKidor Modified